bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -119,9 +119,7 @@ if (dataframe)
 endif()
 
 # SOFIE-GNN pythonizations
-# TODO: GNN tests are right now hardcoded to never run, because it is not clear
-# if they will pass on the new CI. Please revert this.
-if (FALSE AND tmva)
+if (tmva)
     if(NOT MSVC OR CMAKE_SIZEOF_VOID_P EQUAL 4 OR win_broken_tests)
         find_python_module(sonnet QUIET)
         find_python_module(graph_nets QUIET)
```

Comment:
Even re-enabling it on the CMakeLists still leaves this buggy way of finding the python module

Reply:
What do you mean by that? I didn't experience this `find_python_module` function to be buggy

Reply:
Ah you probably mean that it makes it non-deterministic which tests are run